### PR TITLE
Update china holidays

### PR DESCRIPTION
**Pull request by pwojt**

### CODE REVIEW

File: spec/acceptance_cn_spec.rb [hunk 1]
```diff
@@ -2,53 +2,21 @@
 
 describe "CN holidays" do
   %w(
-    2014-01-01
```
Comment (umtrey) [inline spec/acceptance_cn_spec.rb]: Why remove historical holidays? I'd say keep them! The holidays of the past still matter.

Comment (pwojt) [inline spec/acceptance_cn_spec.rb]: That was a business request, I'm not sure why. I'll ask.

Comment (rnubel) [inline spec/acceptance_cn_spec.rb]: I agree with the ghostly shade of @umtrey on this one.

Comment (umtrey) [inline spec/acceptance_cn_spec.rb]: :heart:

Comment (cakephemist) [inline spec/acceptance_cn_spec.rb]: @umtrey: Vade retro satana!

Comment (umtrey) [inline spec/acceptance_cn_spec.rb]: hehe

**Comment by umtrey** [comment]
LGTM, +1

Miss y'all.

File: business_calendar.gemspec [hunk 2]
```diff
@@ -22,6 +22,8 @@ Gem::Specification.new do |spec|
 
   spec.add_development_dependency "bundler", "~> 1.3"
   spec.add_development_dependency "rake"
+  spec.add_development_dependency "pry"
+  spec.add_development_dependency "pry-debugger"
   spec.add_development_dependency "rspec"
```
Comment (rnubel) [inline business_calendar.gemspec]: Can you specify this to be rspec 3.0+? The new expect syntax is, IIRC, new to RSpec 3.

**Pull request merged**